Source/WebKit/NetworkProcess/webrtc/NetworkRTCTCPSocketCocoa.mm

```diff
@@ -49,14 +49,14 @@
 
 using namespace WebCore;
 
-static dispatch_queue_t tcpSocketQueue()
+static const RetainPtr<dispatch_queue_t>& tcpSocketQueue()
```

Comment:
The right way to fix this is to rename this function to `tcpSocketQueueSingleton()`, we don't need to ref a singleton.